test/functional/test_site/testAntiFOUCStyles.md

```diff
@@ -0,0 +1,45 @@
+<!-- Anti-FOUC styles should be applied to navbar and dropdown in navbar. -->
+
+<navbar placement="top" type="inverse">
+  <li><a href="/" class="nav-link">One</a></li>
+  <li><a href="/" class="nav-link">Two</a></li>
+  <dropdown text="Dropdown" class="nav-link">
+    <li><a class="dropdown-item" href="/">Dropdown One</li>
+    <li><a class="dropdown-item" href="/">Dropdown Two</li>
+  </dropdown>
+</navbar>
+
+<!-- Anti-FOUC styles should be applied to dropdown. -->
+# Test dropdown in body with text and class attributes
+
+<dropdown text="Test One" class="test-class">
+  <li><a class="dropdown-item" href="/">Dropdown One</li>
+  <li><a class="dropdown-item" href="/">Dropdown Two</li>
+</dropdown>
+
+<!-- Anti-FOUC styles should be correctly applied to dropdown with no text or class. -->
+
+# Test dropdown in body without text and class attributes
+
+<dropdown>
+  <button slot="button" type="button" class="btn dropdown-toggle">
+    Test Two
+    <span class="caret"></span>
+  </button>
+  <li><a class="dropdown-item" href="/">Dropdown One</li>
+  <li><a class="dropdown-item" href="/">Dropdown Two</li>
+</dropdown>
+
+<!-- Filler text to increase page length. -->
+
+# Filler text
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ultrices gravida dictum fusce ut placerat orci nulla. Tincidunt praesent semper feugiat nibh. Aliquam sem fringilla ut morbi tincidunt augue interdum velit. Vitae aliquet nec ullamcorper sit amet risus nullam eget. Vitae semper quis lectus nulla at volutpat diam. Eu mi bibendum neque egestas congue quisque egestas diam in. Mauris pharetra et ultrices neque ornare aenean euismod elementum. Purus gravida quis blandit turpis cursus in hac habitasse platea. Quis enim lobortis scelerisque fermentum dui faucibus in. Sem fringilla ut morbi tincidunt augue interdum velit euismod. Duis tristique sollicitudin nibh sit. Dignissim suspendisse in est ante in nibh mauris cursus mattis.
```

Comment:
Instead of using text, you can use `<br/>` to increase the page length.

```html
# Filler text

<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>

Some text
```

Reply:
Fixed! 🙂